snmp/datadog_checks/snmp/data/default_profiles/cisco.yaml

```diff
@@ -2,11 +2,4 @@
 
 extends:
   - _base_cisco.yaml
-
-metadata:
-  device:
-    fields:
-      vendor:
-        value: "cisco"
```

Comment:
note: this is redundant since `_base_cisco.yaml` is already importing the vendor field.